core/node/house_keeper/src/fri_witness_generator_jobs_retry_manager.rs

```diff
@@ -88,6 +88,28 @@ impl FriWitnessGeneratorJobRetryManager {
         );
     }
 
+    pub async fn requeue_stuck_recursion_tip_jobs(&mut self) {
+        let stuck_jobs = self
+            .pool
+            .connection()
+            .await
+            .unwrap()
+            .fri_witness_generator_dal()
+            .requeue_stuck_recursion_tip_jobs(
+                self.processing_timeouts.recursion_tip(),
+                self.max_attempts,
+            )
+            .await;
+        let job_len = stuck_jobs.len();
```

Comment:
nit: why not put it directly to metrics? I don't see that you are using this variable anywhere else